src/main/java/uk/gov/hmcts/reform/draftstore/service/crypto/CryptoService.java

```diff
@@ -0,0 +1,85 @@
+package uk.gov.hmcts.reform.draftstore.service.crypto;
+
+import com.google.common.base.Strings;
+import com.google.common.primitives.Bytes;
+import uk.gov.hmcts.reform.draftstore.utils.IllegalArgument;
+
+import java.security.GeneralSecurityException;
+import java.security.MessageDigest;
+import java.security.NoSuchAlgorithmException;
+import java.security.SecureRandom;
+import java.util.Arrays;
+import javax.crypto.Cipher;
+import javax.crypto.SecretKey;
+import javax.crypto.spec.GCMParameterSpec;
+import javax.crypto.spec.SecretKeySpec;
+
+import static com.google.common.base.Charsets.UTF_8;
+
+public class CryptoService {
+
+    private static final String CIPHER = "AES/GCM/NoPadding";
+    private static final Length KEY_LENGTH = Length.BITS_128;
+    private static final Length IV_LENGTH = Length.BITS_96;
+    private static final Length GMC_TAG_LENGTH = Length.BITS_128;
+
+    public static byte[] encrypt(String input, String password) {
```

Comment:
Is there a library that could this for us and be less risky for us to implement?

Reply:
There's not much going on here, we just provide configuration for `javax.Crypto` (key length, IV length, AES mode etc).

The only custom things is that IV is concatenated with cipher text.